### DISCUSSION TITLE
vscode freeze

### DISCUSSION THREAD

**Discussion by akbar-ardiansyah** [community/community#19563] [Other Feature Feedback, Questions, & Ideas]
I'm having a strange thing in the code editor when opened css file so many lines. the editor still freeze.
please provide a solution for those who have experienced,
sorry for bad english

**Comment by melvincwng** [comment]
@akbar-ardiansyah, can you upgrade your VSCode editor to the latest version 1.68?

Hope this resolves your issue.

If this is helpful, kindly 'Mark as answer' and consider following me on GitHub. Would love to expand my GitHub network of developers.

Thank you and have a nice day 😊
